meson.build

```diff
@@ -188,12 +188,19 @@ polkit_agent_dep = dependency('polkit-agent-1', version : '>=0.98', required : g
 build_system_helper = polkit_agent_dep.found()
 
 fuse_dep = dependency('fuse3', version : '>=3.1.1', required : false)
+fusermount = get_option('system_fusermount')
 
 if fuse_dep.found()
   fuse_api = 31
+  if fusermount == ''
```

Comment:
We could still use `find_program()` to ensure it's always an absolute path.

Reply:
I don't know whether we want to force it to always be an absolute path? The trade-off is that if we use an absolute path, then we'll be immune to #5104, but if distros or users move `fusermount` between directories (`bin` vs. `sbin`, rootfs vs. `usr`, etc.) then they will have to recompile Flatpak to pick up that change.

Reply:
I don't have a super strong opinion but I would lean towards absolute paths just for reproducibility.

The `find_program()` calls will default to an absolute path though. So one could argue it would be consistent but one could like that `system_fusermount` is an escape hatch to that behavior.

Reply:
> The find_program() results will default to an absolute path though.

Mm, true. Perhaps that's for the best? That means we'll be immune to #5104, at the cost that people who expect to be randomly moving executables around will have to specify `-Dsystem_fusermount=fusermount`.

Reply:
Fine with me.